qlty-config/src/sources/git_source.rs

```diff
@@ -211,10 +212,12 @@ impl GitSource {
         self.fetch(repository, &mut origin, branches)
     }
 
-    fn fetch(&self, repository: &Repository, origin: &mut Remote, branches: &[&str]) -> Result<()> {
+    fn fetch(&self, _repository: &Repository, origin: &mut Remote, branches: &[&str]) -> Result<()> {
```

Comment:
This `_repository` argument is unused so we should remove it (and make sure the code still typechecks)